.stylelintrc

```diff
@@ -4,13 +4,14 @@
     "stylelint-config-palantir/sass.js"
   ],
   "rules": {
+    "declaration-empty-line-before": null,
```

Comment:
The only thing I'm not a fan of with the current behavior is how it disallows empty lines after `@include` declarations (i.e. `react-transition`). Does this rule address that?